api/issues_tiles.py

```diff
@@ -191,7 +191,7 @@ def issues_mvt(db, z, x, y, format):
             response.content_type = 'application/vnd.mapbox-vector-tile'
             return tile
         else:
-            return HTTPError(404)
+            return HTTPResponse(status=204, headers={'Access-Control-Allow-Origin': '*'})
```

Comment:
Should do the job.
```suggestion
            return HTTPResponse(status=204)
```

Reply:
That was what I thought, but in fact it doesn't (tried with local frontend setup) : if you use HTTPResponse, CORS headers aren't added automatically. If using HTTPResponse, you have either to set this explicitly, or use some module for Bottle (?). Source for this : https://stackoverflow.com/questions/53640565/returning-httpresponse-object-in-bottle-loses-response-headers

Reply:
Ok.